source/admin/config.yml

```diff
@@ -1,6 +1,5 @@
 backend:
   name: git-gateway
-  branch: master
```

Comment:
Removed this because it has no impact for Git Gateway.

Reply:
👍